clang/lib/AST/Interp/EvaluationResult.cpp

```diff
@@ -122,19 +123,22 @@ static bool CheckFieldsInitialized(InterpState &S, SourceLocation Loc,
   }
 
   // Check Fields in all bases
-  for (const Record::Base &B : R->bases()) {
+  for (unsigned I = 0, E = R->getNumBases(); I != E; ++I) {
+    const auto &B = *R->getBase(I);
```

Comment:
```suggestion
  for (auto [I, B] : llvm::enumerate(R->bases())) {
```
Maybe I wasn’t too clear about this, but what I had in mind was using `llvm::enumerate` here and then `std::next` below (which you already did). Is there a reason why this doesn’t work that I’m missing here?

Reply:
Thanks, it's make sense.